system-tests/services/test-external-volumes.js

```diff
@@ -0,0 +1,62 @@
+require("../_support/utils/ServicesUtil");
+
+describe("Services", function() {
+  /**
+   * Test the pods
+   */
+  describe("External Volumes", function() {
+    afterEach(() => {
+      cy.window().then(win => {
+        win.location.href = "about:blank";
+      });
+    });
+
+    it("should be persistent after suspension in single container services", function() {
+      cy.visitUrl(
```

Comment:
in this case both visitUrl calls are shortcuts. I agree that this testcase could be split up, but as it is written for now its only that one block. when we add more to that file it might be a good idea to move it though.